src/rabbit_vhost.erl

```diff
@@ -42,7 +46,7 @@ recover() ->
     %% So recovery will be run every time a vhost supervisor is restarted.
     ok = rabbit_vhost_sup_sup:start(),
 
-    [ok = rabbit_vhost_sup_sup:init_vhost(VHost) || VHost <- rabbit_vhost:list()],
+    [ok = rabbit_vhost_sup_sup:init_vhost(VHost) || VHost <- rabbit_vhost:list_names()],
```

Comment:
I suppose the compiler would see that, but we don't need to specify `rabbit_vhost:` here as it is a local function call.